ray-operator/controllers/ray/common/pod.go

```diff
@@ -689,13 +689,6 @@ func setMissingRayStartParams(rayStartParams map[string]string, nodeType rayv1.R
 	// Add --block option. See https://github.com/ray-project/kuberay/pull/675
 	rayStartParams["block"] = "true"
 
-	// Add dashboard listen port for RayService.
-	if _, ok := rayStartParams["dashboard-agent-listen-port"]; !ok {
```

Comment:
Since we use port 52365 for liveness/readiness probes, perhaps we should always set this Ray start parameter.

```
if _, ok := rayStartParams["dashboard-agent-listen-port"]; !ok {
	rayStartParams["dashboard-agent-listen-port"] = strconv.Itoa(utils.DefaultDashboardAgentListenPort)
}
```

Reply:
Done 54dd0ef